layouts/joomla/form/field/textarea.php

```diff
@@ -42,12 +45,23 @@
  * @var   array    $options         Options available for this field.
  * @var   array    $inputType       Options available for this field.
  * @var   string   $accept          File types that are accepted.
+ * @var   boolean  $charcounter     Does this field support a character counter?
  */
 
+ // Initialize some field attributes.
+if ($charcounter)
+{
+	// Load the js file
+	Factory::getDocument()->getWebAssetManager()->enableAsset('short-and-sweet');
+	// Set the css class to be used as the trigger
+	$charcounter = 'charcount';
```

Comment:
```suggestion
	$charcounter = ' charcount';
```
I think later below in line 64 we need a space between `$class` and `$charcounter`, but only if $charcounter is not boolean false, that's why the space should not be added in line 64.

Reply:
Or does `$class` already have a space at the end? If so, then forget my suggestions.